feature-libs/tracking/tracking.module.ts

```diff
@@ -0,0 +1,10 @@
+import { ModuleWithProviders, NgModule } from '@angular/core';
+
+@NgModule({})
+export class TrackingModule {
```

Comment:
What's the reason for the existence of this module?

Reply:
I guess the idea was to group all the tracking features here.
Removed.